### ISSUE TITLE
Hyperparams for Imagenet

### ISSUE THREAD

**Issue by ykarmesh** [khurramjaved96/mrcl#9]
Hi Khurram, thanks for making the repo available to us. I have been trying to run the repo for the Imagenet dataset and I haven't been able to replicate your results. I think it might be an issue with the hyperparameters that I am using. Will it be possible for you to share the correct set of hyperparameters?

**Comment by khurramjaved96** [comment]
Parameters used:

"params": {
        "commit": false,
        "dataset": "imagenet",
        "dataset_path": "/localscratch/khurram.1977403.0/miniimagenet/",
        "meta_lr": 0.0001,
        "name": "imagenet/0_0001/IMAGENET_0_1_no_reset",
        "no_reset": true,
        "rln": 6,
        "seed": 10000,
        "seeds": [
            10
        ],
        "steps": 26000,
        "tasks": 1,
        "update_lr": 0.1,
        "update_step": 10
    },

**Comment by khurramjaved96** [comment]
I also uploaded the pre-trained oml model for imagenet in the GDrive folder.
